src/main/java/frc/robot/subsystems/jointSubsystem/JointSubsystem.java

```diff
@@ -0,0 +1,69 @@
+package frc.robot.subsystems.jointSubsystem;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.geometry.Rotation2d;
+import frc.robot.hardware.motor.ControllableMotor;
+import frc.robot.hardware.request.IRequest;
+import frc.robot.hardware.signal.InputSignal;
+import frc.robot.subsystems.GBSubsystem;
+
+public class JointSubsystem extends GBSubsystem {
+
+	private final ControllableMotor motor;
+	private final JointCommandsBuilder commandsBuilder;
+	private final InputSignal<Rotation2d> positionSignal;
+	private final InputSignal[] otherSignals;
+	private final IRequest<Rotation2d> positionRequest;
+
+	public JointSubsystem(
+		String logPath,
+		ControllableMotor motor,
+		IRequest<Rotation2d> positionRequest,
+		InputSignal<Rotation2d> positionSignal,
+		InputSignal... otherSignals
+	) {
+		super(logPath);
```

Comment:
consider adding postion into the other signals and refresh all at once (for your choice to do it or not)

Reply:
yeah but i specifically need the position for stayinplave and isatposition

Reply:
u can save it in both palces

Reply:
wdym?

Reply:
save it as seperate signal but also in the array